mainhub/mainhub.ino

```diff
@@ -93,19 +117,63 @@ void update_game()
     cursor_index = 1; // Move to the second LED
     direction = 1; // Change the direction to right
   }
-  // Increment the cursor, but reset to zero if it is greater than the number of leds (Circle Path)
-  /*cursor_index = cursor_index + 1;
-  if (cursor_index >= NUM_LEDS)
-  {
-    cursor_index = 0;
-  }
-*/
+  //LEDs
+  //Light the cursor
+  leds[cursor_index] = rainbowColors[currentLevel];
```

Comment:
There is nothing wrong with this code, but as it starts to grow it might get harder to follow. 

Seperating the updating of variables (update_game) and the drawing of lights (draw) would help to keep things easier to read.

Not an issue yet, but keeping things readable will become a problem soon.